resources/js/components/presenters/party/BuildingsSection.vue

```diff
@@ -0,0 +1,279 @@
+                        {{ $t('Any') + ' ' + building.damageCost }}
+                        <add-links-and-icons class="materials flex ml-1" text="{LUMBER} {METAL} {HIDE}"/>
+                    </div>
+                    <button v-if="!building.isWrecked()" @click="wreck(building)" class="mdc-button mdc-button--raised">
+                        <add-links-and-icons text="{WRECKED}"/>
+                        {{ $t('Wreck') }}
+                    </button>
+                    <button v-else @click="repair(building)" class="mdc-button mdc-button--raised h-auto p-1">
+                        <add-links-and-icons text="{REPAIR}"/>
+                        <div class="bg-dark-background inline-grid grid-cols-3 gap-px ml-1">
+                            <add-links-and-icons text="{LUMBER}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{METAL}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{HIDE}" class="outline-gray p-1"/>
+                            <template v-for="(cost, i) in building.wreckedCost">
+                                <div :key="i" class="outline-gray text-center">{{ cost }}</div>
+                            </template>
+                        </div>
+                    </button>
+                </div>
+            </div>
+            <h3 class="mb-2">{{ $t('Available') }}</h3>
+            <div v-for="building in availableBuildings" :key="building.key()" class="grid xs:grid-cols-3 lg:grid-cols-5 gap-px mb-3">
+                <div class="text-gray-400 outline-gray flex items-center p-3 bg-dark-background">
+                    {{ building.id }} {{ $t(building.name) }} {{ $t('Lvl.') }} 0
+                </div>
+                <div class="xs:col-span-2 outline-gray flex items-center justify-center p-2 bg-dark-background">
+                    <button @click="upgrade(building)" class="mdc-button mdc-button--raised h-auto p-1">
+                        <add-links-and-icons text="{UPGRADE}"/>
+                        <div class="bg-dark-background inline-grid grid-cols-5 gap-px ml-1">
+                            <add-links-and-icons text="{PROSPERITY}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{LUMBER}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{METAL}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{HIDE}" class="outline-gray p-1"/>
+                            <add-links-and-icons text="{COINS}" class="outline-gray p-1"/>
+                            <template v-for="(cost, i) in building.nextUpgradeCost.slice(0, 5)">
+                                <div :key="i" class="outline-gray text-center">{{ cost }}</div>
+                            </template>
+                        </div>
+                    </button>
+                </div>
+            </div>
+            <h3 class="mb-2">{{ $t('Walls') }}</h3>
```

Comment:
I think we can hide the heading if all walls are build, or so a message to indicate it.